packages/remme/src/index.ts

```diff
@@ -404,29 +411,30 @@ namespace Remme {
          * ```
          */
         public constructor(clientInit: IClientInit = {
-            privateKeyHex: "",
+            accountConfig: DEFAULT_ACCOUNT_CONFIG,
             networkConfig: DEFAULT_NETWORK_CONFIG,
         }) {
             let {
                 networkConfig = DEFAULT_NETWORK_CONFIG,
             } = clientInit;
 
             const {
-                privateKeyHex = "",
+                accountConfig = DEFAULT_ACCOUNT_CONFIG,
```

Comment:
how you provide default values to subobject? you should move this line to above operation and then do like specified in 425 line